src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -125,7 +136,13 @@ public async Task ThresholdExceeded_ThrowsException(int? maxResponseHeadersLengt
                     headers.Add(new HttpHeaderData($"Custom-{i}", new string('a', 480)));
                 }
 
-                await server.HandleRequestAsync(headers: headers);
+                try
+                {
+                    await server.HandleRequestAsync(headers: headers);
+                }
+                // Ignore errors caused by the client
```

Comment:
I suggest being more descriptive why are we ignoring these
```suggestion
                // Client can respond by closing/aborting the underlying stream while we are still sending the headers, ignore these exceptions 
```